MdePkg/Test/Mock/Library/GoogleTest/MockPeiServicesLib/MockPeiServicesLib.cpp

```diff
@@ -0,0 +1,23 @@
+/** @file MockPeiServicesLib.cpp
+  Google Test mocks for PeiServicesLib
+  Copyright (c) 2022, Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+#include <GoogleTest/Library/MockPeiServicesLib.h>
+
+MOCK_INTERFACE_DEFINITION(MockPeiServicesLib);
+MOCK_FUNCTION_DEFINITION(MockPeiServicesLib, PeiServicesLocatePpi, 4, EFIAPI);
+MOCK_FUNCTION_DEFINITION(MockPeiServicesLib, pei_GetVariable, 6, EFIAPI);
+
+  
+// Normally PPIVariableServices is "found"
+// This will be defined INSIDE the test, with its definition pointing to the mock function getVariable
+static EFI_PEI_READ_ONLY_VARIABLE2_PPI peiReadOnlyVariablePpi = {

```

Comment:
Why do we have variable service here? I thought this is a PeiServicesLib mock?

Reply:
Similarly to the mock of getVariable in the MockUefiRuntimeServisesTableLib, I'm mocking the functions PeiServices would retrieve. 
Would you suggest doing a mock of peiReadOnlyVariablePpi  in variable services?